crates/cxx-qt-build/src/lib.rs

```diff
@@ -361,6 +365,26 @@ impl CxxQtBuilder {
     pub fn new() -> Self {
         let mut qt_modules = HashSet::new();
         qt_modules.insert("Core".to_owned());
```

Comment:
should we continue always adding `Core` or should this become explicit ? As you likely need to specify it on the CMake side?

Reply:
This should be explicit.
Basically, move this `insert` into the `else` branch of the `if let` below.
If we're getting stuff from CMake, make sure it's exactly as specified by CMake.